cadquery/assembly.py

```diff
@@ -94,6 +94,11 @@ class Assembly(object):
     objects: Dict[str, "Assembly"]
     constraints: List[Constraint]
 
+    # Allows metadata to be stored for exports
```

Comment:
I'd propose to mark those items as "private" for now, i.e. `_subshape_names`, etc